UI/Login.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=
+    , initial-scale=1.0">
+    <title>login/registration</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="header">
+        <h1>LOGIN</h1>
+        <div class="form">
+            <form action="#" method="post">
+                <div class="txt-field">
+                    <input type="text" required>
+                    <span></span>
+                    <label for="#">Username</label>
```

Comment:
If the label is not being used for a particular text field, remove it. consider reading about how ti use labels